spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/cache/CacheAutoConfiguration.java

```diff
@@ -134,4 +142,45 @@ public void afterPropertiesSet() {
 
 	}
 
+	/**
+	 * Exception thrown when {@link org.springframework.cache.CacheManager} implementation
+	 * are not specified.
+	 */
+	static class NoCacheManagerException extends RuntimeException {
```

Comment:
To mirror `NoSuchBeanDefinitionException`, you could call that `NoSuchCacheManagerException`